src/window/glfw/Cargo.toml

```diff
@@ -28,6 +28,9 @@ documentation = "https://docs.rs/gfx_window_glfw"
 name = "gfx_window_glfw"
 
 [dependencies]
-glfw = "0.13"
+glfw = "0.14"
 gfx_core = { path = "../../core", version = "0.7" }
 gfx_device_gl = { path = "../../backend/gl", version = "0.14" }
+
+[dev-dependencies]
+gfx = "0.15"
```

Comment:
actually, you need to provide both the local path and the version, similar to the dependencies above